### PR TITLE
Style/abd style cleanup

### PR DESCRIPTION
**Pull request by Abdeboskey**
# Som Timer Pull Request Form

## Description
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

This branch was all about style. This push features many final style cleanups/polishes, including:
* adding some gradients to the header, background, and buttons
* normalizing shadows and borders
* adjusting margins and sizing
* adjusting icons/assets
* editing prompts and error text
* and other minor tweaks here and there

  This branch also fixed a bug where the video player was mysteriously off-center, and where the background was cutting off and not filling up the view when scrolling.

  This branch also introduced a function that when a mood is selected, will drain the color and opacity from the mood Icons that are not selected.  

  Finally, this branch addressed many of the linter errors that were being shown in the console, and updated tests to reflect breaking changes.

## Type of change
- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Style
## Has This Been Tested?
- [ ] No
- [x] Yes
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration:
* This branch is being pushed with all tests passing and has been visually inspected in the browser with the eyes, and the devtools.

# Checklist:
- [x] My code has no unused/commented out code
- [x] My code has no console logs
- [x] I have performed a self-review of my own code
- [x] I have commented on my code, particularly in hard-to-understand areas

moodIcons

### CODE REVIEW

File: src/Components/About/About.js [hunk 1]
```diff
@@ -7,30 +7,103 @@ const About = () => {
...
       <section className={style.contributors}>
         <section className={style.engineerList}>
           <h3>Front End Engineers: </h3>
-          <p><a href="https://github.com/Abdeboskey" target="_blank" rel="noopener noreferrer">Aaron Burris-DeBoskey</a></p>
-          <p><a href="https://github.com/jkwest-93" target="_blank" rel="noopener noreferrer">Jake West</a></p>
-          <p><a href="https://github.com/rwilliams659" target="_blank" rel="noopener noreferrer">Rachel Williams</a></p>
+          <p>
+            <a
+              href="https://github.com/Abdeboskey"
+              target="_blank"
```
Comment (rwilliams659) [inline src/Components/About/About.js:48]: Thanks for cleaning up these warnings we were getting for not having `rel="noopener noreferrer"`

File: src/Components/App/App.test.js [hunk 2]
```diff
@@ -48,7 +48,7 @@ describe('App', () => {
     const statsIcon = getByAltText('stats icon')
     fireEvent.click(statsIcon)
 
-    const statsHeading = getByRole('heading', { name: /⏳ stats page coming soon! ⏳/i })
+    const statsHeading = getByRole('heading', { name: /stats page coming soon!/i })
```
Comment (rwilliams659) [inline src/Components/App/App.test.js:51]: Glad we were able to get the tests passing again without too much trouble. That was certainly an adventure!

File: src/Components/ContentDelivery/ContentDelivery.module.scss [hunk 3]
```diff
@@ -11,50 +11,57 @@
...
+  @include size(42em, 21.4em);
+  display: flex;
+  justify-content: center;
 }
 
 .videoPlayer {
   @include borderAndShadow(none, 1em, 0px 0px 20px 2px $midBlue);
   background-color: $midBlue; 
+  background: radial-gradient($midBlue, #7987a1ab);
+  box-shadow: 0px 0px 20px $darkBlue, inset 0px 0px 5px rgba(0, 0, 0, 0.239);
```
Comment (rwilliams659) [inline src/Components/ContentDelivery/ContentDelivery.module.scss:27]: Love the consistent box shadows throughout the site; really gives it a unified feel

File: src/Components/MoodRating/MoodRating.js [hunk 4]
```diff
@@ -37,6 +38,17 @@ const MoodRating = () => {
     }
   }
 
+  const selectFace = event => {
```
Comment (rwilliams659) [inline src/Components/MoodRating/MoodRating.js:41]: Adding this functionality was next-level - can't wait to demo it!

File: src/Components/MoodRating/MoodRating.js [hunk 5]
```diff
@@ -127,8 +140,8 @@ const MoodRating = () => {
             {error && <p className={style.errorMessage}>
               <span role='img' alt='Error sign'>
                 ⚠
-              </span>
-              Please select an option above to continue! 
+              </span>{" "}
+              Please select one of the faces above to continue!{" "}
```
Comment (rwilliams659) [inline src/Components/MoodRating/MoodRating.js:144]: Nice catch making this error message more specific - I think small changes like that really add to the user experience

File: src/Components/Stats/Stats.js [hunk 6]
```diff
@@ -4,12 +4,26 @@ import style from './Stats.module.scss'
 const Stats = () => {
   return (
     <article>
-      <h2 className={style.prompt}>⏳ Stats Page Coming Soon! ⏳</h2>
+      <h2 className={style.prompt}>
+        <span role="img" alt="hourglass emoji" aria-label="hourglass emoji">
```
Comment (rwilliams659) [inline src/Components/Stats/Stats.js:8]: Wow, didn't know this was the standard for emojis - good to know moving forward!

**Review by rwilliams659 — APPROVED** [review]
Awesome job cleaning all this up Aaron!!

File: src/Components/App/App.test.js (hunk 2 above)
Comment (Abdeboskey) [reply src/Components/App/App.test.js:51]: Twas an adventure indeed!!! Thanks for all the help 🙏🏼

File: src/Components/ContentDelivery/ContentDelivery.module.scss (hunk 3 above)
Comment (Abdeboskey) [reply src/Components/ContentDelivery/ContentDelivery.module.scss:27]: note for the future, these shadows would be a great addition to the `variables` file.

**Review by Chulstro — APPROVED** [review]
You're all champions, this looks amazing

File: src/Components/App/App.test.js (hunk 2 above)
Comment (jkwest-93) [reply src/Components/App/App.test.js:51]: We <3 version control!!!

File: src/Components/App/App.test.js [hunk 7]
```diff
@@ -144,7 +144,7 @@ describe('App', () => {
 
     fireEvent.click(skipIcon)
 
-    const moodRatingHeading = getByRole('heading', { name: /please select a face below/i })
+    const moodRatingHeading = getByRole('heading', { name: /how are you feeling right now?/i })
```
Comment (jkwest-93) [inline src/Components/App/App.test.js:147]: I really like this adjustment to the prompt!! This wording seems to invite more introspection!

File: src/Components/ContentSelection/ContentSelection.js [hunk 8]
```diff
@@ -10,8 +10,8 @@ import meditationContentIcon from '../../assets/content/meditationContentIcon.pn
 import yogaContentIcon from '../../assets/content/yogaContentIcon.png'
 
 const ContentSelection = () => {
-  const [ view, setView ] = useContext(ViewContext)
-  const [ videoLink, setVideoLink ] = useContext(VideoContext)
+  const setView = useContext(ViewContext)[1]
```
Comment (jkwest-93) [inline src/Components/ContentSelection/ContentSelection.js:13]: Thanks for updating these & clearing out other terminal errors!

**Review by jkwest-93 — APPROVED** [review]

**Pull request merged**